dev/vagrant/conf/README.md

```diff
@@ -28,5 +28,5 @@ There are two primary cases for customization.
 
 ### Add pre and post bootstrap tasks
 
-- [pre_bootstrap](pre_bootstrap/README.md)
-- [post_bootstrap](post_bootstrap/README.md)
+- [pre_bootstrap](../pre_bootstrap/README.md)
+- [post_bootstrap](../post_bootstrap/README.md)
```

Comment:
Broken links, I guess these related to https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/65

Reply:
Yes, related. That's why I didn't fix it since on next sync it should be fixed already in the template.